docs/standard/attributes/retrieving-information-stored-in-attributes.md

```diff
@@ -37,29 +38,29 @@ Retrieving a custom attribute is a simple process. First, declare an instance of
  [!code-csharp[Conceptual.Attributes.Usage#18](../../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.attributes.usage/cs/source3.cs#18)]
  [!code-vb[Conceptual.Attributes.Usage#18](../../../samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.attributes.usage/vb/source3.vb#18)]  
   
- This program displays the following text when executed.  
+ This program displays the following text when executed:
   
 ```console  
 The Name Attribute is: Joan Smith.  
 The Level Attribute is: 42.  
 The Reviewed Attribute is: True.  
 ```  
   
- If the attribute is not found, the **GetCustomAttribute** method initializes `MyAttribute` to a null value. This example checks `MyAttribute` for such an instance and notifies the user if no attribute is found. If the `DeveloperAttribute` is not found in the class scope, the following message displays to the console.  
+ If the attribute isn't found, the **GetCustomAttribute** method initializes `MyAttribute` to a null value. This example checks `MyAttribute` for such an instance and notifies the user if no attribute is found. If the `DeveloperAttribute` isn't found in the class scope, the console displays the following message:
   
 ```console  
 The attribute was not found.
 ```  
   
- This example assumes that the attribute definition is in the current namespace. Remember to import the namespace in which the attribute definition resides if it is not in the current namespace.  
+ The preceding example assumes that the attribute definition is in the current namespace. Remember to import the namespace in which the attribute definition resides if it isn't in the current namespace.  
```

Comment:
There seems to be an inconsistency in the usage... Other places it is "previous"

Reply:
In this context I would either keep "This" or use "preceding" as it is in this update -- "previous" could be interpreted to mean an earlier example.